submissions/Jemikalajah-Philip/Html/homepage.html

```diff
@@ -35,10 +35,10 @@
                         <img src="icon/logo.png" width="80px" height="80px">
                     </div>
                     <div class="navbar">
-                        <a href="#">Home</a>
-                        <a href="#">Help</a>
+                        <a href="homepage.html">Home</a>
+                        <a href="#" >Help</a>
                         <a href="#">Contact</a>
-                        <a href="#">Men</a>
+                        <a href="#" onclick="displayMenProduct()" id="prod">Men</a>
```

Comment:
Using `onclick` is actually bad practice. Entangles your html with your js. 
Please use the `addEventListener` DOM API. @see https://www.w3schools.com/js/js_htmldom_eventlistener.asp